ssz-rs/src/lib.rs

```diff
@@ -64,14 +93,27 @@ where
     T::deserialize(encoding)
 }
 
-#[derive(Debug, Error)]
-#[error("{0}")]
+#[derive(Debug)]
 pub enum SimpleSerializeError {
-    Serialize(#[from] SerializeError),
-    Deserialize(#[from] DeserializeError),
-    Merkleization(#[from] MerkleizationError),
-    List(#[from] ListError),
-    Vector(#[from] VectorError),
+    Serialize(SerializeError),
+    Deserialize(DeserializeError),
+    Merkleization(MerkleizationError),
+    List(ListError),
+    Vector(VectorError),
+}
+
+impl Display for SimpleSerializeError {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        match self {
+            SimpleSerializeError::Serialize(error) => write!(f, "serialize error: {}", error),
+            SimpleSerializeError::Deserialize(error) => write!(f, "deserialize error: {}", error),
+            SimpleSerializeError::Merkleization(error) => {
+                write!(f, "merkleization error: {}", error)
+            }
+            SimpleSerializeError::List(error) => write!(f, "list error: {}", error),
+            SimpleSerializeError::Vector(error) => write!(f, "vector error: {}", error),
```

Comment:
looks like we can replace all of these with `write!(f, "{error}"),

to retain the existing error formatting

Reply:
I ended up removing these as they aren't necessary anymore.